src/monodroid/monodroid.props

```diff
@@ -3,12 +3,13 @@
   <PropertyGroup>
     <_DebugCFlags>-ggdb3 -O0 -fno-omit-frame-pointer</_DebugCFlags>
     <_ReleaseCFlags>-g -O2</_ReleaseCFlags>
-    <_CommonCFlags>-Ijni -Ijni/zip "-I$(MonoSourceFullPath)\mono\eglib" -std=c99 -DSGEN_BRIDGE_VERSION=$(MonoSgenBridgeVersion) -D_REENTRANT -DHAVE_CONFIG_H -DMONO_DLL_EXPORT -DJI_DLL_EXPORT -fno-strict-aliasing -ffunction-sections -fvisibility=hidden -Wformat -Werror=format-security</_CommonCFlags>
+    <_CommonCFlags>-Ijni -Ijni/zip "-I$(MonoSourceFullPath)\mono\eglib" -std=c99 -DSGEN_BRIDGE_VERSION=$(MonoSgenBridgeVersion) -D_REENTRANT -DHAVE_CONFIG_H -DMONO_DLL_EXPORT -DJI_DLL_EXPORT -DDYLIB_MONO -fno-strict-aliasing -ffunction-sections -fvisibility=hidden -Wformat -Werror=format-security</_CommonCFlags>
     <_HostUnixCFlags>$(_CommonCFlags) -Wa,--noexecstack</_HostUnixCFlags>
     <_HostUnixLdFlags>-Wall -lstdc++ -lz -shared -fpic</_HostUnixLdFlags>
     <_HostCommonWinCFlags>$(_CommonCFlags) -DWINDOWS -DNTDDI_VERSION=NTDDI_VISTA -D_WIN32_WINNT=_WIN32_WINNT_VISTA -fomit-frame-pointer</_HostCommonWinCFlags>
     <_HostCommonWinLdFlags>-Wall -lstdc++ -lz -shared -fpic -ldl -lmman -pthread -lwsock32 -lole32 -luuid -lshlwapi -lpsapi</_HostCommonWinLdFlags>
     <_UnixAdditionalSourceFiles>$(MonoSourceFullPath)\support\nl.c jni\debug.c jni\monodroid-networkinfo.c jni\xamarin_getifaddrs.c</_UnixAdditionalSourceFiles>
+    <_HostAdditionalSourceFiles>$(JavaInteropSourceDirectory)\src\java-interop\java-interop-gc-bridge-mono.c $(JavaInteropSourceDirectory)\src\java-interop\java-interop.c $(JavaInteropSourceDirectory)\src\java-interop\java-interop-mono.c</_HostAdditionalSourceFiles>
```

Comment:
These files shouldn't be limited to "the host"; they should be present in *all* versions, i.e. part of `@(_CFile)`.

Reply:
I didn't want to increase the device runtime size yet. I have added it to all runtimes then. Let see if it works.

On desktop the `jnimarshalmethod-gen.exe` crashes now, hopefully I can fix it today.